Build troubles with CODESIGN.Submitter

Hi.

I can't build any firmware solution neither sdk since I get always following errors I can't spot what could depend on.

I uploaded full log at https://dl.dropboxusercontent.com/u/7414592/TinyCLR.it/msbuild_log.zip but it seems that root problem is (please forgive italian msbuild log):

error CS0246: Il nome di tipo o di spazio dei nomi 'CODESIGN' non è stato trovato. Probabilmente manca una direttiva using o un riferimento all'assembly. [D:\NETMF\netmf-interpreter\Framework\Tools\BuildTasksInternal\BuildSigner\BuildSignerSpotBuild.csproj]

i.e. cannot find "CODESIGN.Submitter" assembly...

...any hint?

Thanks!

--- a/Framework/Tools/BuildTasksInternal/BuildSigner/BuildSignerSpotBuild.csproj
+++ b/Framework/Tools/BuildTasksInternal/BuildSigner/BuildSignerSpotBuild.csproj
@@ -35,6 +35,7 @@
     <Reference Include="CODESIGN.Submitter, Version=4.1.0.0, Culture=neutral, PublicKeyToken=3d8252bd1272440d, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>$(MSBuildProgramFiles32)\Microsoft Internal\Codesign.Submitter\CODESIGN.Submitter.dll</HintPath>
+      <HintPath>$(SPOROOT)\tools\x86\CODESIGN\CODESIGN.Submitter.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Xml" />

